docs/sources/enterprise/usage-insights/presence-indicator.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Presence indicator"
+description = "Know who is looking at the same dashboard as you"
+keywords = ["grafana", "presence-indicator", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/presence-indicator.md"]
+weight = 300
++++
+
+# Presence indicator
```

Comment:
```suggestion
# See who is present
```

Reply:
I think we should not change this until we change it everywhere.